homeassistant/components/sensibo/select.py

```diff
@@ -77,37 +77,35 @@ def __init__(
         super().__init__(coordinator, device_id)
         self.entity_description = entity_description
         self._attr_unique_id = f"{device_id}-{entity_description.key}"
-        self._attr_name = (
-            f"{coordinator.data.parsed[device_id]['name']} {entity_description.name}"
-        )
+        self._attr_name = f"{self.device_data.name} {entity_description.name}"
 
     @property
     def current_option(self) -> str | None:
         """Return the current selected option."""
-        return self.device_data[self.entity_description.remote_key]
+        return getattr(self.device_data, self.entity_description.remote_key)
```

Comment:
You could potentially abstract away the need to do `getattr` and `setattr` calls by moving them behind a function in the library

Reply:
Shouldn't that be with the coordinator instead as the writing and reading is from coordinator data and what would be the benefit compared to use `getattr` and `setattr`?